pyproject.toml

```diff
@@ -40,7 +40,7 @@ log_cli = true
 source = ["src", "*/site-packages"]
 
 [tool.coverage.run]
-source = ["cockpit"]
+source_pkgs = ["cockpit"]
```

Comment:
Please squash this commit ("use source_pkgs for coverage")

Reply:
Yup.